src/layout/Footer.js

```diff
@@ -6,20 +6,35 @@ const Footer = () => {
 	const year = new Date().getFullYear();
 	const iconStyle = "cursor-pointer";
```

Comment:
You can just remove this and put the style in-line to the Social icons.

Reply:
Done!